### ISSUE TITLE
 Warning: setState(...): Can only update a mounted or mounting component. 

### ISSUE THREAD

**Issue by chetstone** [Purii/react-native-tableview-simple#21]
I was debugging a tricky problem in my code when I saw this warning:

```
 Warning: setState(...): Can only update a mounted or mounting component. This usually means you called setState() on an unmounted component. This is a no-op. Please check the code for the Cell component.
```
I can make that warning go away by passing `allowFontScaling={false}` to all my Cell components. Then the Cell component doesn't try to set state. This also makes my other problem go away. I have no idea why.

This is an almost-acceptable workaround, because I have been debating whether to turn off font scaling in my settings pages. If users have the scaling set too high, it can make a mess. But I see Apple uses font scaling in their Settings app, so I guess best practice is to use it.

The other problem with the workaround is that in some cells I use a RightDetail, and the detail text is still being scaled, resulting in an ugly cell with two different font sizes, like this.

<img width="347" alt="screen shot 2016-12-02 at 15 41 21" src="https://cloud.githubusercontent.com/assets/976343/20852903/e360fca0-b8a5-11e6-8911-98d87397bd1d.png">

**Comment by Purii** [comment]
Thank you!

I will look into it. Seems to be related to [Cell.js:45](https://github.com/Purii/react-native-tableview-simple/blob/master/components/Cell.js#L45).
Also will fix the issue with the font size 👍

**Comment by Purii** [comment]
Should be fixed with [v0.14.1](https://github.com/Purii/react-native-tableview-simple/releases/tag/v0.14.1). :+1: 
_Just reopen the issue, if your problem is still not fixed._
